libs/metrics/top_level_metrics.py

```diff
@@ -46,6 +46,7 @@ class MetricsFields(common_fields.ValueAsStrMixin, str, enum.Enum):
     INFECTION_RATE_CI90 = "infectionRateCI90"
     ICU_CAPACITY_RATIO = "icuCapacityRatio"
     BEDS_WITH_COVID_PATIENTS_RATIO = "bedsWithCovidPatientsRatio"
+    WEEKLY_COVID_ADMISSIONS_DENSITY_RATIO = "weeklyCovidAdmissionsPer100k"
```

Comment:
nit: For clarity I'd be a fan of sticking to "per 100k" terminology instead of "density ratio".  So I would just do `WEEKLY_COVID_ADMISSIONS_PER_100K`

I know Igor did WEEKLY_CASE_DENSITY_RATIO instead of WEEKLY_CASES_PER_100K but since that's directly mirroring the existing CASE_DENSITY_RATIO I think it's okay.

Reply:
Yeah, that sounds good to me! I'll update